electrophysiology/models.py

```diff
@@ -106,21 +118,26 @@ class SpikeSortedUnit(BaseExperimentalData):
     channel_group = models.IntegerField(null=True, blank=True,
                                         help_text="which shank this unit came from (an integer not a link)")
     trough_to_peak_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, computed from unfiltered mean waveform. TODO: which waveform?")
+                                 help_text="ms, computed from unfiltered mean spike waveform.")
     half_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, half width of negative peak in unfiltered waveform. TODO: which waveform?")
+                                 help_text="ms, half width of negative peak in unfiltered spike waveform.")
     trough_to_peak_amplitude = models.FloatField(null=True, blank=True,
-                                 help_text="µV, from filtered waveform. TODO: which waveform?")
+                                 help_text="µV, from filtered spike waveform.")
+								 # when you say which waveform, do you mean which channel? I guess the peak, but we can leave it unspecified...
     refractory_violation_rate = models.FloatField(null=True, blank=True,
-                                 help_text="fraction of spikes occurring < 2ms. TODO: is 2ms going to be hard-coded?")
+                                 help_text="fraction of spikes occurring < 2ms. ")
+								 # TODO: is 2ms going to be hard-coded?
+								 # i guess... what's the alternative?
```

Comment:
The point is that the user might want to do a search over this, to find only good cells with a low violation rate.

That is a complicated query with an array field, right?